controller/src/beerocks/master/son_management.cpp

```diff
@@ -1535,6 +1535,51 @@ void son_management::handle_bml_message(Socket *sd,
         }
         break;
     }
+    case beerocks_message::ACTION_BML_WIFI_CREDENTIALS_UPDATE_REQUEST: {
+        LOG(TRACE) << "ACTION_BML_WIFI_CREDENTIALS_UPDATE_REQUEST";
+        uint32_t ret = 0;
+        auto request =
+            beerocks_header
+                ->addClass<beerocks_message::cACTION_BML_WIFI_CREDENTIALS_UPDATE_REQUEST>();
+        if (request == nullptr) {
+            LOG(ERROR) << "addClass cACTION_BML_WIFI_CREDENTIALS_UPDATE_REQUEST failed";
+            return;
+        }
+
+        auto al_mac = network_utils::mac_to_string(request->params());
+
+        auto agent_mac = database.get_node_parent_ire(al_mac);
+
+        if (!agent_mac.empty()) {
+            auto cmdu_header =
+                cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_RENEW_MESSAGE);
+
```

Comment:
The RENEW message requires 3 TLVs:
- AL-MAC of the controller;
- SupportedRole (= controller);
- SupportedFreqBand.

So you have to add these.

For the freqband, you can include both for now, with a TODO to only include the bands that changed.